src/redux/initialState.js

```diff
@@ -1,17 +1,39 @@
-const initialState = {
-  reduxTokenAuth: {
-    currentUser: {
-      isLoading: false,
-      isSignedIn: false,
-      attributes: {
-        firstName: null,
-        lastName: null,
-        dateOfBirth: null,
-        profileImage: null,
-        uid: null,
-      },
+export const defaultState = {
+  currentUser: {
+    isSignedIn: false,
+    attributes: {
+      firstName: null,
+      lastName: null,
+      dateOfBirth: null,
+      profileImage: null,
+      uid: null,
+    },
+    headers: {
+      'access-token': null,
+      'token-type': 'Bearer',
+      client: null,
+      expiry: null,
+      uid: null,
     },
   },
 };
 
-export default initialState;
+export const initialState = {
+  currentUser: {
+    isSignedIn: false,
+    attributes: {
+      firstName: null,
+      lastName: null,
+      dateOfBirth: null,
+      profileImage: null,
+      uid: null,
+    },
+    headers: {
+      'access-token': null,
+      'token-type': 'Bearer',
+      client: null,
+      expiry: null,
+      uid: null,
+    },
+  },
+};
```

Comment:
I see that defaultState is the same exactly as initialState can we just keep one?!